lib/usd/utils/tests/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+find_package(GTest REQUIRED)
+
+set(USDUTILS_TEST_EXECUTABLE_NAME AL_USDUtilsTests)
+set(USDUTILS_TEST_NAME GTest:AL_USDUtilsTests)
+set(USDUTILS_PYTHON_TEST_NAME Python:AL_USDUtilsTests)
+
+add_definitions(
+  -DAL_EXTRAS_TEST_DATA="${CMAKE_CURRENT_SOURCE_DIR}/data"
+)
+
```

Comment:
Use of add_definitions is prohibited.   use `target_compile_definitions ` instead and explicitly declare the keyword entries.

Reply:
We don't bury unit tests inside subdirectories inside `maya-usd/lib`. All relevant unit tests should go into `maya-usd/test`.